src/app/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { useEffect } from "react";
+import Routing from "pages/routes/ui/routes";
+import { withProviders } from "./providers";
+import { onAppMounted } from "entities/session";
+import "./index.css";
+
+const App = () => {
+  useEffect(() => {
+    onAppMounted();
```

Comment:
event should be named from the model side/view (what happened related to model)

Reply:
exported as model and changed the occurence